cdap-security/src/test/java/co/cask/cdap/security/authorization/InMemoryAuthorizer.java

```diff
@@ -80,23 +87,33 @@ public void enforce(EntityId entity, Principal principal, Set<Action> actions) t
       }
     }
     if (!allowedForRoles.containsAll(actions)) {
-      throw new UnauthorizedException(principal, Sets.difference(actions, allowed), entity);
+      throw new UnauthorizedException(principal, Sets.difference(actions, allowed), entity, true);
     }
   }
 
   @Override
   public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
-    // TODO: needs to be implemented
-    throw new UnsupportedOperationException("This method needs to be implemented!");
+    if (superUsers.contains(principal) || superUsers.contains(allSuperUsers)) {
```

Comment:
We dont have super users now, but this is left to provide a way to bypass some auth check for some specific users. For example, in https://github.com/caskdata/cdap/blob/release/4.3/cdap-cli-tests/src/test/java/co/cask/cdap/cli/AuthorizationCLITest.java#L96, we skip the check for grant/revoke, I think it can be left here to provide a convenient way to bypass some checks.